src/quest/QuestList.cpp

```diff
@@ -0,0 +1,84 @@
+#include "QuestList.h"
+
+QuestList::QuestList() : CYIAbstractDataModel(1)
+{
+    
+}
+
+QuestList::~QuestList()
+{
+    
+}
+
+QuestList* QuestList::FromJSON(CYIString path)
+{
+    yi::rapidjson::Document* document;
+    CYIParsingError parsingError;
+
+    document = CYIRapidJSONUtility::CreateDocumentFromFile(path, parsingError);
+    YI_ASSERT(!parsingError.HasError(), "QuestList::FromJSON", parsingError.GetParsingErrorMessage());
+
+    return FromJSON(document);
+}
+
+QuestList* QuestList::FromJSON(yi::rapidjson::Document* document)
+{
+    CYIParsingError parsingError;
+
+    yi::rapidjson::Value& questArray = (*document)["Quests"];
+    YI_ASSERT(questArray.IsArray(), "QuestList::FromJSON", "Could not find quests array in JSON file.");
+
+    QuestList* questList = new QuestList();
+
+    for (yi::rapidjson::SizeType i = 0; i < questArray.Size(); ++i) 
+    {
+        const yi::rapidjson::Value& questObject = questArray[i];
+
+        questList->AddQuest(QuestModel::FromJSON(questObject), i);
+    }
+
+    return questList;
+}
+
+CYIString QuestList::Display()
```

Comment:
i would rename this to ToString since it is generating a string and not really displaying it.